gst-plugin-pravega/src/pravegasink/imp.rs

```diff
@@ -219,6 +266,8 @@ enum State {
         // The offset that will be written to the index upon end-of-stream.
         final_offset: Option<u64>,
         buffers_written: u64,
+        tx: Sender<()>,
+        handle: Option<JoinHandle<()>>,
```

Comment:
Rename `handle` to `retention_thread_handle`.

Reply:
changed.